src/NuGet.Core/NuGet.Protocol/Plugins/PluginManager.cs

```diff
@@ -304,8 +304,15 @@ private async Task<Lazy<IPluginMulticlientUtilities>> PerformOneTimePluginInitia
             {
                 throw new ArgumentNullException(nameof(pluginFactoryCreator));
             }
-
-            _rawPluginPaths = reader.GetEnvironmentVariable(EnvironmentVariableConstants.PluginPaths);
+#if IS_DESKTOP
+            _rawPluginPaths = reader.GetEnvironmentVariable(EnvironmentVariableConstants.DesktopPluginPaths);
+#else
+            _rawPluginPaths = reader.GetEnvironmentVariable(EnvironmentVariableConstants.CorePluginPaths);
+#endif
```

Comment:
If we make our core libraries, like `NuGet.Protocol` only target `netstandard2.0` in the future, and let the .NET Framework use the .NET Standard versions, then this will stop working. We need a runtime check to see if we're on the .NET Framework on .NET Core.

Reply:
NuGet.Protocol depends on Packaging, so it will keep cross targeting. 
But yeah we'll need to account for conditional compilation in the projects that will switch to only 1 framework.

Reply:
I'd argue separation of concerns :) 
It's a problem for when we try to reduce our frameworks. 
Note that I'm 100% confident we won't need to do that with Plugins and NuGet.Protocol is higher on the dependency graph where .NET Framework APIs are just required due to Credentials needing them.